### PR TITLE
release highlights for v0.50.0

### PR DESCRIPTION
**Pull request by Aniket-Engg**

### CODE REVIEW

File: releaseDetails.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
 {
-    "version": "v0.49.0",
+    "version": "v0.50.0",
     "title": "RELEASE HIGHLIGHTS",
-    "highlight1": "Syntax highlighting for .toml files added",
-    "highlight2": "'Deploy & Run Transactions' plugin's UI improved",
-    "highlight3": "Remix AI model improved",
+    "highlight1": "Pin a plugin on right side",
```
Comment (yann300) [inline releaseDetails.json:4]: `Pin a plugin on the right side` ?

Comment (Aniket-Engg) [reply releaseDetails.json:4]: We can move a plugin on right side

Comment (Aniket-Engg) [reply releaseDetails.json]: ok, updated

Comment (ryestew) [reply releaseDetails.json]: New right side panel for plugins
Remix UI available in 한국인

Comment (ryestew) [reply releaseDetails.json]: New right side panel for plugins

Comment (ryestew) [reply releaseDetails.json]: Pin a plugin to the new right side panel

**Review by ryestew — APPROVED** [review]

**Pull request merged**